hack/ci/start-kind.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+#     Copyright 2020 Nexus Operator and/or its authors
+#
+#     This file is part of Nexus Operator.
+#
+#     Nexus Operator is free software: you can redistribute it and/or modify
+#     it under the terms of the GNU General Public License as published by
+#     the Free Software Foundation, either version 3 of the License, or
+#     (at your option) any later version.
+#
+#     Nexus Operator is distributed in the hope that it will be useful,
+#     but WITHOUT ANY WARRANTY; without even the implied warranty of
+#     MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#     GNU General Public License for more details.
+#
+#     You should have received a copy of the GNU General Public License
+#     along with Nexus Operator.  If not, see <https://www.gnu.org/licenses/>.
+
+set -e
```

Comment:
I think `set -e` is fine in this script, but be careful with it, it may not behave as you expect in [certain situations](http://mywiki.wooledge.org/BashFAQ/105).